src/VCR/Util/StreamProcessor.php

```diff
@@ -280,15 +284,16 @@ public function stream_tell()
      *
      * @link http://www.php.net/manual/en/streamwrapper.url-stat.php
      *
-     * @param  string $path The file path or URL to stat.
+     * @param string $path  The file path or URL to stat.
+     * @param int    $flags Holds additional flags set by the streams API.
      *
-     * @return integer      Should return as many elements as stat() does.
+     * @return integer Should return as many elements as stat() does.
      */
-    public function url_stat($path)
+    public function url_stat($path, $flags)
     {
         $this->restore();
         try {
-            $result = @stat($path);
+            $result = @stat($path, $flags);
```

Comment:
`stat` has no second argument, so we need to do that work ourselves it seems. no idea what we really have to do.